app/modules/entities/views/editor/properties_editor.js

```diff
@@ -36,3 +36,13 @@ export default Marionette.CompositeView.extend({
     this.ui.showAllProperties.hide()
   }
 })
+
+const dataadminOnlyShortlistedProperties = [
+  'wdt:P31'
+]
+
+const getPropertiesShortlistPerRole = propertiesShortlist => {
```

Comment:
i think by role, you mean access level, since admin too  will have access to this property